.github/extendJSON/resolve.py

```diff
@@ -29,10 +29,14 @@ def resolve_extended_json_file(file_data_dict):
     resolve = resolve_extended_json_file(superJsonDict)
 
     resolve['rules'].update(file_data_dict)
+    resolve['rules'].pop('axioms')
```

Comment:
Extra fields were being populated during the merge of the dictionaries. The presence of these fields broke the configuration parser.